Project/MyGameLibrary/InventoryObject.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Fall2020_CSC403_Project.code;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyGameLibrary
+{
+    public abstract class InventoryObject
+    {
+        
+        protected bool stackable;  // Should multiples of this type stack in a single inventory slot?
+        protected bool exhaustible;  // Can this object be used more than once?
+        protected int count;  // How many of this object are in this stack (always 1 if not stackable).
+        public Image img; // This will be the inventory icon for this inventory item.
+
+        // Returns how many of this object are in the stack
+        public int GetCount()
+        {
+            return count;
+        }
+
+        // Adds addCount to the count
+        public void Add(int addCount)
+        {
+            count += addCount;
+        }
+
+        // Removes removeCount from the count.
+        // Returns if the operation was successful.
+        public bool Remove(int removeCount)
+        {
+            if(removeCount > GetCount())
+            {
+                return false;
+            }
+            count -= removeCount;
+            return true;
+            
+        }
+
+        // Returns if this object is stackable
+        public bool IsStackable()
```

Comment:
Instead of creating methods like this to access these fields, it would make more sense for them to be treated as properties for the object. This statement applies to all of the IsWhatever variables in this class as well as to the get count. Consider this
```
protected bool _stackable; 
public bool Stackable { get => stackable; }
```
Now later on instead of addressing the values as 
```
iObj.IsStackable()
iObj.IsExhaustible()
iObj.GetCount()
```
you can instead just use
```
iObj.Stackable
iObj.Exhaustible
iObj.Count
```